x-pack/plugins/security_solution/public/common/store/inputs/model.ts

```diff
@@ -68,12 +68,13 @@ export interface GlobalGraphqlQuery extends GlobalGenericQuery {
   id: string;
   refetch: null | Refetch | RefetchWithParams;
 }
+
 export interface GlobalKqlQuery extends GlobalGenericQuery {
   id: 'kql';
   refetch: RefetchKql;
 }
 
-export type GlobalQuery = GlobalGraphqlQuery | GlobalKqlQuery;
+export type GlobalQuery = GlobalKqlQuery | GlobalGraphqlQuery;
```

Comment:
yea, do we need the `GraphqlQuery` part anymore?

Reply:
i dont think so, but lots of type changes come with removing it so i will do it in the follow up type pr